src/Stormpath/Util/UserAgent.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Stormpath\Util;
+
+class UserAgent {
```

Comment:
This is a good idea for a more OO approach. However, if we want to go this rout I would actually do it differently:

1. I would rename this class to `UserAgentBuilder` because this class is really for building a user agent string.
2. Instead of having an `add` method, I would have "setters" for each property that would set the array value and the calling component wouldn't have to know about the key.
3. Have a `build` method that builds the user agent string.